### ISSUE TITLE
[Sybil Report] 123 wallets

### ISSUE THREAD

**Issue by SybilBucket** [LayerZero-Labs/sybil-report#141]
# Reported Addresses
```
0x8932aD51fF66F062f1BeeA9cF018C1aA62e612cb				
0x09d6899D92066fb8b1a975c657649d695A262D74				
0xA83a1c94938B99a71B871b38dD7e4F026d43F6F4				
0xF508b826b5b64AA762177b01685DFEED4a3a2eA8				
0x00C5b20952dF9B62f26a3e0d3c48c81c5636d9cA				
0xfa896a06c4049D6105Aa4B085Cb370790EEBc8dF				
0x18BBdd0fc89cF9F728703b41abf65063155FA84D				
0x142f18CE8597a471a0FEC8aE29B31EBF4983999b				
0xE3A4db1d36072De9A0b34F4F911A3B56900a58A6				
0xa47d23423D00efc614CF422498ED65114e527fb6				
0xD67AAE19463fbc8420f70c281b6EE8E466f90899				
0xabE0bB2f45e648C1cc704146Dfb6DC6c9c7Dc15C				
0xc160e4628b649A801fED48E2b022e7d31B9b7299				
0x1688Adff4d7009e50df2d78A278C12731Dad534B				
0x6C2F08BA3AEC357335Fc543eB6b1F42613c4A5aB				
0x9d57Cbf61573749672c1D7d9635D49fa2598F3e1				
0x68330898Ae40ab8AB5d2F6Fb4a5bbF18be6dEf7D				
0x3F749050A154F2356C5C5d528D05E44EaF57E71F				
0x280f17b7e3f4D9c16A947cD845D510D6f4286860				
0xead59977E816696B0D47694Fa6860404F8619DE7				
0xC9a0f3fe8b528f8dfd59999EAD62f847d8a05994				
0xc9b157f3E901c7411bc88f2bB7F703F2F0D81784				
0x15d6034b7471C625702d628040C178Bf7B85d300				
0xD39EEcDf70A7e3bBCd3DAD1d1bF7EEEaDcE61c72				
0x7f0AEBD6d086E7D027503E8Ad126F9f9BCfE13E1				
0xEFc330a5d6C00e69C40FaD48D83278D5E8EB0B7F				
0x6B76a1028c0d525022c8247ffe0d186434f14C5a				
0x0d2c393a0429c03AB58D9c80071667c8F880A125				
0xB6CBf1A8fcFB69d946c71498EBFE0a40001C31f9				
0x7fa9f94159a416a1157A1d7A0D707c517d4D3f72				
0x7C8cb665F842ddC37c270c5572368cD76cb94f33				
0xBc7c4E5E8fD74376d7095fB5757Af646592E2939				
0x137FE7915C963206B933D97e7F4549664c8C2Cff				
0xc776af4cd7D4c3CA0951EA5eDE5a5Cf385AC13Bb				
0x4e65fB9d25a3e4Bc5701Fe4Bf557c0dCd8f54dA2				
0xA8b032Bd75Ea2664b6d2ffbA1726fd41972d2883				
0xC669c64B3ea21Ff1527B110bEFa3fFC0bb3422a8				
0x7f478A4bc8C1BC0957d31E5563258d3e2AA562C4				
0x870b344Fb5237CC9D1d918354cE5b5AB98453e8a				
0xf68b6E96AA9FA37efB603f779E11698874fCa5dD				
0xe760596d9AcEC7DA943ee61ccbd6AD766D31346C				
0x18094d014412efec1F47AF49c813d6f4673f1566				
0x5BEd4b3cCf1550c1df286646035f178f20c7c3bf				
0xfCCb7893D1f35fc2683060F91eB7f0e8003A8431				
0xC94f6CB8a2e998bC61367C434809330c1Aac6cFD				
0x7b203184E9E348D3CcefF8d8deFe13C3a01d962d				
0x4B0Eb9C0b00fb792CE65070872df4d198520a88F				
0x6710c72558DD85CdE01d13D7ca87257a26354456				
0x516624804dBFd300954E9dB8852437CD734f6485				
0x64d9167836B722208b3DFFEB450D54bA49741201				
0x01c8698C358305B1cdcd413E60BBdeE83C9D474a				
0x4AfF155C1e27329D571725FD8dD15b6F96208ea3				
0xa6946cb0408f60f0336791a69912A43BdE3a9C5f				
0x4EC0f37086CC6F7A1932fce9790c61b4D3B86D61				
0x7b0E5797f8Ce902D1a366208B1922f4dDCd11b62				
0x3138a573621F0eA968385E284dD248B4862Bd1D5				
0x3A7d5E7eA27cD248A45EAD0aA2d7f5E2D3622A67				
0x43098C6b62F15C339880B6a66d9762D635eDD8d7				
0xdF8699C94018A8489e1E2A5e59f449c217671564				
0x20Cc7805F508a4e1c12F8E0BD3e7fD2Fc55C6DBA				
0x80fc6cAe5A2F7dbbD423d9271078B9Bc2287bf2e				
0x8f010d290f3Ab748F6948515cd839a9f84D3A858				
0xEc62d7a654418f738d032b950E9aD041D86A0629				
0xA0d1212F3CD1Ff8Ee5366F29865d8aA6F44ab982				
0x85494655205d0D3F788a97d57B59b5808a1897CE				
0x989cfA5EFbDA94a3D24250C00Fa228EFEa877970				
0x9d211735d36BF06215728F07390C79D02F637d11				
0x77871C284Ff4ce49dA4517a178a91dd37BbC40B4				
0xaa6A6bCbbd1aea667420D42e3d30c2Ebb9CC31Ed				
0x8EE9Cdc28c5038759aAC44652A8EC78D9EA48229				
0xcE317EBAaA7AeC68cf18B611eb5cFCBA953ee207				
0xC31d0F0CB1086cBe7d3d8C058fD84c9803b5434D				
0x4BF0C1b3fC9A550cAf378b245be92D297eD64955				
0x8E78199857f16010770430f84220050C80B75E84				
0x60821770e298Dbb3BF617EeA86738eE3962A2e75				
0x39a73e94916Db0162Ff9c48cC511649daEb15dc6				
0x87C2aAb8bF129B05BACdd13752beBEfB4167843f				
0x4CfA7A1D5AF597d76b914920edDEC79670A9141D				
0x5b318d1c979676091E7b111bec5d9d500D14Cd52				
0x9731ae6BcA613faEDf51F37a2036e804fd88B831				
0x4E3E88c7eD9Ec50f9ecb505E434D18C11702436C				
0xC362Ed676A119Bfd976aA2CE87f6e9274d533B5f				
0xe351c71eEE799362795005D2B4e492286EfcAb8b				
0xaF7D3B9A4e39CC52FAC61E3dB4de836276125c51				
0x95CE55e0D21C928385b6E7b5867e52DC8B7EA24B				
0xDE809b362a21df3f4e023038c6AFc4D82533D929				
0xB93ffFD14d45a8D53e6A12a68C9fd1aeC5d3161d				
0x0692266a81BBcB3793D0ea6907d0B0bb90599db1				
0x041Aff702f9e074E555e88060b326E2280827baD				
0x77fe38f8C6fDe65746777Db91172840F90cA5742				
0x24FA70Eb63aF7991AE191856C753014c5dbdc885				
0xC5Dc23A8A7B13fFFfbA8EBae9A5A7F886389D1dE				
0x045530d455dd2494C807E9DC27d611c2Cb6F9A52				
0x515eE0B9141DaE5469386e5be3743264a22A1971				
0xBD124f68f51DB3c831E142177e0C1f2a40eeA53A				
0x044436Db116C583071151979dd0645F61a49dEce				
0x012527A00E91F41b3dB8b64cf4fC1a1d8AECCd96				
0x7E5E000584835D4a2D194795e9F9Ad31df9C6A16				
0xeaBe5386F73D9EF4A673bf0edBb36cb4E359D64c				
0x005Bb287Dbf885Bcf360c87c92CAEaa8e0fD7E14				
0xaaaCD194bBfAB0bc2d07394ac543F6717aD0Bf45				
0x2DED5C7d11Dd41f9C0d80C9aa07862339995795b				
0xEdC765e3d4A74BDC95f8b177327DFF701075Eb71				
0x652F6886BB065d60d5a99f74D73fBcCA7c1aA6da				
0x78da1e663F50bC5f6dB2B3d34739624FC17F651f				
0x7a4eEA981e1e8255Aa76E152Cce976122633b2aD				
0x09e68B2D290825b62e97a07968EBa1170BA746E6				
0x7519015d91DC4d279D2A0A04fB3519627E52Ca70				
0x874A900dC4929870F3e1965a60FdD3A26AA8D302				
0xD4B148231B15918490e308Bf37c11E972993F3df				
0x26dA2c4EAde9c6ae81271839F0bca5A386a06202				
0x11066A8534d69A93623922dD87ed4106bf468C46				
0x7e469Fa4006CD2F704567B54B8925A391aF60b7D				
0xeEF6e31EC9183C60b1494052498d38FdEc2097f3				
0x0718056FABB983e1E17D2ce17aF9b31021B62c66				
0x1d7c5Da33362321712d286C670fe99ca83114D79				
0xFcEBD90877A7EA3A12610332C537ACAdBCA7be10				
0xDfd8303d8344774dB8c1F38D89e769ceEE5DdE9A				
0xb8d15f85e96a5bb2fFa22016f3E70eb90Ca660a0				
0x739e5496e69e3FD0551183c9A5d88d345FC11890				
0x534a391c5FF111792a22066dfc1F19413C328a67				
0xf9545e07D38B77C2BEFD2783EACaa014077cFc70				
0x77468E9a02bdc92B9614c6ba6a763deD5A06d8d2				
```
# Description
### You have a much more advanced toolkit in partnership with Nansen & Chaos than I have. So PLEASE accept my report, I've spent so much time putting it all together and searching it. It definitely can't be an organic user, with so many intersections and same actions.

# Detailed Methodology & Walkthrough
### ARKHAM + DEBANK + SCRIPTS

I wrote a script that searches for wallets with a balance on Arbitrum of up to $3 and any number of staked $STG.  All of them received ETH from the exchange (most often Binance), after which they returned some of it back (to activate the wallet on the Ethereum or L2s)  and began sending ETH from Arbitrum to Optimism, then bought $STG on Slingshot and lock them. Then they received money on the ZkSync network and continued to farm volumes there. Since these accounts are very similar, I assume that these may be accounts run by the script.

### To summarize: These accounts were registered in the same month, have the same pattern of actions and similar characteristics on nftcopliot.com:
- 10k+ Volume
- 30+ Txs
- 8 Source chains
- 10 Destination chains
- 10-15 Contracts

# Reward Address (If Eligible)
### `0xFA744219F02EA36FAe2cF56BaD87055A77034720`
